apps/web/components/RedirectToLastSolved.tsx

```diff
@@ -10,12 +10,9 @@ export const RedirectToLastSolved = ({ trackId }: { trackId: string }) => {
   const router = useRouter();
 
   useEffect(() => {
-    const problemId = getFirstProblemForTrack(trackId);
-    try {
+    getFirstProblemForTrack(trackId).then((problemId) => {
```

Comment:
@N4r35h you can add a catch condition here.

Reply:
a catch condition would not be required, as the getFirstProblemForTrack function itself has a try catch inside it and returns null if the database call fails for some werid reason, in that case we would redirect to /<track>/null and show the error page its not the best but it would be a very rare case